src/components/Person/Person.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+    {age && (<p className="Person__age">{`I am ${age}`}</p>)}
```

Comment:
```suggestion
    {age && (
      <p className="Person__age">
        I am {age}
      </p>
    )}
```
Formatting is important. Now it's hard to read